README.rst

```diff
@@ -1,14 +1,29 @@
 YAUML
 ~~~~~
 
-A script for generating UML diagrams from `YAML <http://www.yaml.org/>`__
-files.
+A script for generating UML diagrams from `YAML <http://www.yaml.org/>`__ files.
+
+
+Dependencies
+============
+
+To run the script, the following softwares are needed :
+
+- Python 3;
+- `PyYAML <https://bitbucket.org/xi/pyyaml>`__ (PyYAML for Python2 and Python3
```

Comment:
While you're at it, you could remove the text in parenthesis. This is an error I've made when I was doing some test long ago. There is no conflict.